scripts/setup_juno_optee.sh

```diff
@@ -0,0 +1,2485 @@
+-# CONFIG_SCHEDSTATS is not set
+-# CONFIG_TIMER_STATS is not set
+-CONFIG_DEBUG_PREEMPT=y
++CONFIG_SCHED_DEBUG=y
++CONFIG_SCHEDSTATS=y
++CONFIG_TIMER_STATS=y
++# CONFIG_DEBUG_PREEMPT is not set
+ 
+ #
+ # Lock Debugging (spinlocks, mutexes, etc...)
+@@ -1671,8 +2479,29 @@
+ CONFIG_EVENT_TRACING=y
+ CONFIG_CONTEXT_SWITCH_TRACER=y
+ CONFIG_TRACING=y
++CONFIG_GENERIC_TRACER=y
+ CONFIG_TRACING_SUPPORT=y
+-# CONFIG_FTRACE is not set
++CONFIG_FTRACE=y
++CONFIG_FUNCTION_TRACER=y
++CONFIG_FUNCTION_GRAPH_TRACER=y
++# CONFIG_IRQSOFF_TRACER is not set
++# CONFIG_PREEMPT_TRACER is not set
++# CONFIG_SCHED_TRACER is not set
++# CONFIG_FTRACE_SYSCALLS is not set
++# CONFIG_TRACER_SNAPSHOT is not set
++CONFIG_BRANCH_PROFILE_NONE=y
++# CONFIG_PROFILE_ANNOTATED_BRANCHES is not set
++# CONFIG_PROFILE_ALL_BRANCHES is not set
++# CONFIG_STACK_TRACER is not set
++# CONFIG_BLK_DEV_IO_TRACE is not set
++# CONFIG_PROBE_EVENTS is not set
++CONFIG_DYNAMIC_FTRACE=y
++# CONFIG_FUNCTION_PROFILER is not set
++CONFIG_FTRACE_MCOUNT_RECORD=y
++# CONFIG_FTRACE_STARTUP_TEST is not set
++# CONFIG_TRACEPOINT_BENCHMARK is not set
++# CONFIG_RING_BUFFER_BENCHMARK is not set
++# CONFIG_RING_BUFFER_STARTUP_TEST is not set
+ 
+ #
+ # Runtime Testing
+@@ -1693,26 +2522,45 @@
+ # CONFIG_SAMPLES is not set
+ CONFIG_HAVE_ARCH_KGDB=y
+ # CONFIG_KGDB is not set
+-# CONFIG_STRICT_DEVMEM is not set
++CONFIG_STRICT_DEVMEM=y
+ # CONFIG_PID_IN_CONTEXTIDR is not set
+ 
+ #
+ # Security options
+ #
+-# CONFIG_KEYS is not set
++CONFIG_KEYS=y
++# CONFIG_PERSISTENT_KEYRINGS is not set
++# CONFIG_BIG_KEYS is not set
++# CONFIG_ENCRYPTED_KEYS is not set
++# CONFIG_KEYS_DEBUG_PROC_KEYS is not set
+ # CONFIG_SECURITY_DMESG_RESTRICT is not set
+ CONFIG_SECURITY=y
+-# CONFIG_SECURITYFS is not set
+-# CONFIG_SECURITY_NETWORK is not set
+-# CONFIG_SECURITY_PATH is not set
+-# CONFIG_SECURITY_SMACK is not set
++CONFIG_SECURITYFS=y
++CONFIG_SECURITY_NETWORK=y
++# CONFIG_SECURITY_NETWORK_XFRM is not set
++CONFIG_SECURITY_PATH=y
++CONFIG_LSM_MMAP_MIN_ADDR=0
++CONFIG_SECURITY_SELINUX=y
++# CONFIG_SECURITY_SELINUX_BOOTPARAM is not set
++# CONFIG_SECURITY_SELINUX_DISABLE is not set
++CONFIG_SECURITY_SELINUX_DEVELOP=y
++CONFIG_SECURITY_SELINUX_AVC_STATS=y
++CONFIG_SECURITY_SELINUX_CHECKREQPROT_VALUE=1
++# CONFIG_SECURITY_SELINUX_POLICYDB_VERSION_MAX is not set
++CONFIG_SECURITY_SMACK=y
+ # CONFIG_SECURITY_TOMOYO is not set
+-# CONFIG_SECURITY_APPARMOR is not set
++CONFIG_SECURITY_APPARMOR=y
++CONFIG_SECURITY_APPARMOR_BOOTPARAM_VALUE=1
++CONFIG_SECURITY_APPARMOR_HASH=y
+ # CONFIG_SECURITY_YAMA is not set
+ # CONFIG_IMA is not set
+ # CONFIG_EVM is not set
+-CONFIG_DEFAULT_SECURITY_DAC=y
+-CONFIG_DEFAULT_SECURITY=""
++# CONFIG_DEFAULT_SECURITY_SELINUX is not set
++# CONFIG_DEFAULT_SECURITY_SMACK is not set
++CONFIG_DEFAULT_SECURITY_APPARMOR=y
++# CONFIG_DEFAULT_SECURITY_DAC is not set
++CONFIG_DEFAULT_SECURITY="apparmor"
++CONFIG_XOR_BLOCKS=y
+ CONFIG_CRYPTO=y
+ 
+ #
+@@ -1720,13 +2568,12 @@
+ #
+ CONFIG_CRY
```

Comment:
Is it possible to have the patch as a separate file so that this script is easier to scroll through and read and not thousands of lines long?